### ISSUE TITLE
**<tab> results in a concatenated list of files, not a fzf menu

### ISSUE THREAD

**Issue by majamin** [junegunn/fzf#1882]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [ ] bash
    - [x] zsh
    - [ ] fish
    
## Problem / Steps to reproduce

st 0.8.2
zsh 5.7.1
fzf 0.20.0

`$ vim **<tab>` produces a list of concatenated files:

```bash
$ vim File1 File2 File3 File4
```

 in the current directory, not a `fzf` menu as desired.

*Sometimes* the menu appears, but I'm not sure when or how this occurs.

**Comment by Wi11Wang** [comment]
I think you may installed through Linuxbrew and forget to install "useful key bindings and fuzzy completion"
You can activate it by using`$(brew --prefix)/opt/fzf/install`.

**Comment by lukastong** [comment]
> I think you may installed through Linuxbrew and forget to install "useful key bindings and fuzzy completion"
> You can activate it by using`$(brew --prefix)/opt/fzf/install`.

it works, thanks

**Comment by majamin** [comment]
Thanks @Wi11Wang. Your comment set me on a different perspective and I managed to fix the problem. It was not a `brew` issue (I don't use Linuxbrew). I originally installed fzf through the arch repo `pacman -S fzf`, which does not ask to enable "useful key bindings, etc.". Not sure how to change that part. Regardless, messy steps, but this fixed it for me:

1. `pacman -S fzf`
2. `cd && git clone https://github.com/junegunn/fzf.git`
3. `cd fzf`
4. `./install`
5. Say _y_ to all questions

The messy part is that now I have the correct config, but a _user_ binary of fzf. I followed up with a few clean up steps on my machine and it works.
